packages/core/src/queries/custom-phrase.ts

```diff
@@ -2,12 +2,21 @@ import { CreateCustomPhrase, CustomPhrase, CustomPhrases } from '@logto/schemas'
 import { sql } from 'slonik';
 
 import { buildInsertInto } from '@/database/insert-into';
-import { convertToIdentifiers } from '@/database/utils';
+import { convertToIdentifiers, manyRows } from '@/database/utils';
 import envSet from '@/env-set';
 import { DeletionError } from '@/errors/SlonikError';
 
 const { table, fields } = convertToIdentifiers(CustomPhrases);
 
+export const findAllCustomPhrases = async () =>
+  manyRows(
+    envSet.pool.query<CustomPhrase>(sql`
+      select ${sql.join(Object.values(fields), sql`,`)}
+      from ${table}
+      order by ${fields.languageKey}
```

Comment:
`order by` seems meaningless at this level, since the client will still have to sort the orders along with built-in languages

Reply:
I think it is nice to have a predictable order at the API level,
no matter whether the client will sort them or not.